AdminCore.DataETL/TypeConverters/TypeToStringConverter.cs

```diff
@@ -0,0 +1,16 @@
+using AdminCore.DataETL.Attributes;
+
+namespace AdminCore.DataETL.TypeConverters
+{
+    /// <summary>
+    /// Converts a type to a string using the ToString() method.
+    /// </summary>
+    public class TypeToStringConverter<T> : ITypeConverter
+    {
+        public object ConvertTo(object value)
+        {
+            string stringValue = ((T) value)?.ToString();
+            return stringValue ?? "";
```

Comment:
I'm missing a lot of context but why return an empty string? This implies the value returned was available which is not true

Reply:
My reasoning is that if a null value is converted that should be represented in the CSV/exported file as an empty string rather than "null".

Reply:
🤔